.dockerignore

```diff
@@ -0,0 +1,6 @@
+.env
+.git
+build
+log
+go.work

```

Comment:
Go sometimes also generates a `go.work.sum` for any time a workspace is missing a sum in the main `go.sum` file. This should also be ignored

Reply:
Added